app/models/miq_report_result.rb

```diff
@@ -350,6 +350,14 @@ def self.with_current_user_groups
     current_user.admin_user? ? all : where(:miq_group_id => current_user.miq_group_ids)
   end
 
+  def self.with_chargeback
+    where(:db => Chargeback::SUBCLASSES)
```

Comment:
STI Loader handles this...we'll have to extend STI Loader to deal with this situation.